### PR TITLE
fix(platform): table last fixed column border fix

### LINKED ISSUE #4415: Table (platform): column freezing borders don't match designs
#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.
1. borders of frozen columns don't align with non-frozen columns
2. border separating frozen and non-frozen columns should be darker per design

#### If this is a bug, please provide steps for reproducing it.
1. go to https://fundamental-ngx.netlify.app/#/platform/table
2. scroll down to the column freezing examples
3. look at the borders between frozen and non-frozen columns

*Actual:*

*Expected:*

### PR DESCRIPTION
**Pull request by platon-rov**
#### Please provide a link to the associated issue.

Closes #4415

#### Please provide a brief summary of this pull request.

[Fix](https://github.com/SAP/fundamental-styles/pull/2384) provided in Fundamental Styles applied.
The last fixed column in the table component has black right border (left in case of RTL).

#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [n/a] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [X] Run npm run build-pack-library and test in external application

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [X] Documentation Examples
- [X] Stackblitz works for all examples

### CODE REVIEW

File: libs/platform/src/lib/components/table/table.component.html [hunk 1]
```diff
@@ -72,6 +72,7 @@
                 [id]="id+'__'+column.name"
                 [attr.aria-colindex]="colIdx"
                 [class.fd-table__cell--fixed]="_freezableColumns?.includes(column.name)"
+                [class.fd-table__cell--fixed-last]="_isLastFreezableColumn(colIdx)"
```
Comment (dimamarksman) [inline libs/platform/src/lib/components/table/table.component.html:75]: can we simply do the check as this: `column.name === freezeColumnsTo`?

Comment (platon-rov) [reply libs/platform/src/lib/components/table/table.component.html]: You're right! Thank you :)

File: libs/platform/src/lib/components/table/table.component.html [hunk 2]
```diff
@@ -72,6 +72,7 @@
                 [id]="id+'__'+column.name"
                 [attr.aria-colindex]="colIdx"
                 [class.fd-table__cell--fixed]="_freezableColumns?.includes(column.name)"
+                [class.fd-table__cell--fixed-last]="column.name === freezeColumnsTo"
```
Comment (kavya-b) [inline libs/platform/src/lib/components/table/table.component.html:75]: don't we need to consume the version of Fundamental Styles where `fd-table__cell--fixed-last` was introduced? I don't see its styles currently: 
image
also the table looks broken

Comment (platon-rov) [reply libs/platform/src/lib/components/table/table.component.html:75]: We do, you just cannot see it on netlify deploy due to the cached npm packages used by netlify (where old fundamental-styles library used).

image

Broken layout has own PR #5614. Text truncation will be fixed in scope of table refactoring PR (it's in progress...).

**Review by kavya-b — APPROVED** [review]

**Review by olshansky — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**